pkg/client-sdk/covenantless_client.go

```diff
@@ -633,6 +633,87 @@ func (a *covenantlessArkClient) Claim(ctx context.Context) (string, error) {
 	return a.selfTransferAllPendingPayments(ctx, pendingVtxos, boardingUtxos, receiver, desc)
 }
 
+func (a *covenantlessArkClient) GetTransactionHistory(ctx context.Context) ([]Transaction, error) {
+	spendableVtxos, spentVtxos, err := a.ListVtxos(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	return vtxosToTxs(spendableVtxos, spentVtxos)
+}
+
+func vtxosToTxs(spendable, spent []client.Vtxo) ([]Transaction, error) {
```

Comment:
we should call this like the following for consistency maybe?
```suggestion
func vtxosToTxsCovenantless(spendable, spent []client.Vtxo) ([]Transaction, error) {
```

Reply:
Done